doc/multi-o.md

```diff
@@ -0,0 +1,117 @@
+# Multi Orchestrator Setup
+
+This document describes the various ways an orchestrator could run multiple nodes and separate concerns. 
+
+## Unlocking Account and Connecting to Ethereum
+
+Unlocking your Ethereum account and connecting to a JSON-RPC provider are necessary steps to run the Livepeer node unless it's in standalone transcoder mode. 
```

Comment:
Maybe use the active voice? "The Livepeer node needs you to unlock your Ethereum account and provide connection information for a JSON-RPC provider, unless the node is in standalone transcoder mode." [1]

Do we have documentation on what connecting to a JSON-RPC provider entails? Maybe link to that, or add a quick blurb if none exists? Also note the slightly modified language here, since the node is the one connecting to the JSON-RPC provider, not the user; the user just needs to provide the connection information.

Also, maybe briefly describe "standalone transcoder" mode, or link if there's a corresponding doc.

Reply:
Also - this section is seems more broadly applicable to all on-chain use cases, rather than multi-O specifically. Probably don't need to refactor the docs right now, but just noting that this part could be extracted into more general-purpose onchain documentation someday.

Reply:
fixed in 4131a02